platform/pulp/platform/models/consumer.py

```diff
@@ -0,0 +1,67 @@
+"""
+Django models related to content consumers.
+"""
+
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db.models import TextField, ForeignKey, ManyToManyField, CASCADE
```

Comment:
By convention (Django convention, at least), this should be `from django.db import models`. I'd like this to be our convention in Pulp as well, unless there are strong objections. This convention can been seen throughout Django's documentation, and it would be good to have our models look as much like "normal" documented Django models as possible.